docs/swagger.yaml

```diff
@@ -396,18 +399,17 @@ components:
           properties:
             giftCard:
               type: object
+              required:
+                - salesPercentage
+                - validityDuration              
               properties:
                 salesPercentage:
                   type: string
                   enum:
-                    - UNDER_TEN
-                    - TEN_OR_MORE
+                    - LT_10
+                    - GTE_10
                   description: How much percentage gift card sales consists of
-                  example: TEN_OR_MORE
-                turnoverShare:
-                  type: string
-                  description: Estimated turnover share of gift card sales in percentage
-                  example: about 25%
+                  example: GTE_10
```

Comment:
Could this be a `boolean`?  If what we need to know is whether the amount is "less than 10" or "ten or more": How about something like a boolean `turnoverShareOverTenPercent`? I think the enums are a bit strange, so even though I don't think the boolean suggestion is super elegant, at least it's a bit simpler..?

Reply:
Agree, 💡